docs/mathematics/lcm-of-two-numbers.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: LCM of two Numbers
+slug: /mathematics/lcm-of-two-numbers
+---
+
+## Introduction
+
+Lowest Common Multiple(LCM) is the lowest number which is divisible by both the numbers. It is also called as Least Common Divisor(LCD). LCM of two numbers cannot be less than the greater of them. If smaller number is the factor of greater number then the greater number will be the LCM.
+
+## Explanation 
+
+LCM of two numbers `a` and `b` is denoted by `lcm(a,b)`. LCM of any number with `1` will be the number itself. LCM of two coprime numbers is the product of numbers. LCM can be found by Listing Multiples, Prime Factorization, Ladder Method, and Divison Method.
+
+LCM has following properties:
+```
+lcm(a,b) = lcm(b,a)
+lcm(da, db, dc) = d * lcm(a, b, c), where d is any positive integer
+lcm(a, b, c) = lcm(lcm(a, b), c) = lcm(a, lcm(b, c))
+```
+
+Let's have one example: 
+```
+num1 = 4
+num2 = 6
+maxx = 6, it contains maximum of num1 and num2
+
+Iteration 1:
+maxx = 6
+maxx is not divisible by both numbers, so increase maxx by 1
+maxx = maxx + 1
+
+Iteration 2:
+maxx = 7
+maxx is not divisible by both numbers, so increase maxx by 1
+maxx = maxx + 1
+
+Iteration 3:
+maxx = 8
+maxx is not divisible by both numbers, so increase maxx by 1
+maxx = maxx + 1
+
+Iteration 4:
+maxx = 9
+maxx is not divisible by both numbers, so increase maxx by 1
+maxx = maxx + 1
+
+Iteration 5:
+maxx = 10
+maxx is not divisible by both numbers, so increase maxx by 1
+maxx = maxx + 1
+
+Iteration 6:
+maxx = 11
+maxx is not divisible by both numbers, so increase maxx by 1
+maxx = maxx + 1
+
+Iteration 7:
+maxx = 12
+maxx is divisible by both the numbers, so this is our LCM
+```
+
+## Algorithm
+
+- Declare a variable `maxx` and initialize it to maximum of two numbers.
+
+- Increase `maxx` by `1` untill it become divisible by both the numbers.
```

Comment:
*until it becomes

Remove extra lines in between.

Reply:
done